pkg/app/pipedv1/controller/scheduler_test.go

```diff
@@ -270,7 +270,7 @@ func TestExecuteStage_SignalTerminated(t *testing.T) {
 
 	handler.Terminate()
 	finalStatus := s.executeStage(sig, s.deployment.Stages[0])
-	assert.Equal(t, model.StageStatus_STAGE_RUNNING, finalStatus)
+	assert.Equal(t, model.StageStatus_STAGE_FAILURE, finalStatus)
```

Comment:
This test assertion is changed due to the previous version of `s.reportStageStatus` use WaitNext, which return no error on context cancel, this should be STAGE_FAILURE as mentioned in this change.
ref: https://github.com/pipe-cd/pipecd/blob/master/pkg/app/pipedv1/controller/scheduler.go#L463